embedchain/embedchain.py

```diff
@@ -97,13 +95,10 @@ def load_and_embed(self, loader: BaseLoader, chunker: BaseChunker, src, metadata
         metadatas = embeddings_data["metadatas"]
         ids = embeddings_data["ids"]
         # get existing ids, and discard doc if any common id exist.
-        where = {"app_id": self.config.id} if self.config.id is not None else {}
-        # where={"url": src}
-        existing_docs = self.collection.get(
-            ids=ids,
-            where=where,  # optional filter
+        existing_ids = self.db.get(
```

Comment:
I didnt get how this part is working.
Also we should not skip where. `where` can have more variables.

Reply:
Update: not skipping where, but currently only using app_id if present in where as filter while fetching from elasticsearch.